ui/selectmenu.js

```diff
@@ -589,11 +590,17 @@ return $.widget( "ui.selectmenu", {
 	},
 
 	_toggleAttr: function() {
-		this.button
-			.toggleClass( "ui-corner-top", this.isOpen )
-			.toggleClass( "ui-corner-all", !this.isOpen )
-			.attr( "aria-expanded", this.isOpen );
-		this.menuWrap.toggleClass( "ui-selectmenu-open", this.isOpen );
+		this.button.attr( "aria-expanded", this.isOpen );
+
+		// We cant use two _toggleClass calls here because we need me make sure
+		// we always remove classes first and add them second or if both classes have the
```

Comment:
Puncuation FTW! Also "otherwise" makes this much more obvious: "we always remove classes first and add them second, otherwise if both classes have the [...]"